spec/support/rails_seven_one.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "active_support/testing/assertions"
+
+module ActiveSupport
+  module Testing
+    module Assertions
+      # Assertion that the block should not raise an exception.
+      #
+      # Passes if evaluated code in the yielded block raises no exception.
+      #
+      #   assert_nothing_raised do
+      #     perform_service(param: 'no_exception')
+      #   end
+      def assert_nothing_raised
```

Comment:
Why did this need patching?

Reply:
"Why?" is unclear, but I get NoMethodError `assert` with rails 7.1